web-services/client/src/main/java/datawave/webservice/model/ModelList.java

```diff
@@ -13,8 +15,23 @@
 @XmlRootElement(name = "ModelList")
 public class ModelList extends BaseResponse implements Serializable, HtmlProvider {
     
+    private String jqueryUri;
+    private String dataTablesUri;
+    private String modelTableName;
+    
     private static final long serialVersionUID = 1L;
-    private static final String TITLE = "Model Names", EMPTY = "";
+    private static final String TITLE = "Model Names";
+    private static final String DATA_TABLES_TEMPLATE = "<script type=''text/javascript'' src=''{0}''></script>\n"
+                    + "<script type=''text/javascript'' src=''{1}''></script>\n"
+                    + "<script type=''text/javascript''>\n"
+                    + "$(document).ready(function() '{' $(''#myTable'').dataTable('{'\"bPaginate\": false, \"aaSorting\": [[0, \"asc\"]], \"bStateSave\": true'}') '}')\n"
+                    + "</script>\n";
+    
+    public ModelList(String jqueryUri, String datatablesUri, String dataWaveMetadateTableName) {
```

Comment:
Shouldn't the last parameter be called `modelTableName`?

Reply:
The models are being stored in the metadata data table is what I am seeing.

Reply:
> 
> 
> The models are being stored in the metadata data table is what I am seeing.

I don't understand this comment. I was pointing out that your parameter `dataWaveMetadateTableName` is misnamed. First, you probably meant it to be `dataWaveMetadataTableName` but second, the parameter should really be called modelTableName because that's what it is (and that is also the member variable name into which you are storing the parameter). The fact that currently in the system we use the datawave metadata table to store model info doesn't mean that will always be the case.